docs/index.rst

```diff
@@ -133,7 +133,21 @@ The following example initializes the extension with the standard Flask command-
     from flask_migrate import Migrate
     migrate = Migrate(app, db)
 
-The two arguments to ``Migrate`` are the application instance and the Flask-SQLAlchemy database instance. The ``Migrate`` constructor also takes additional keyword arguments, which are passed to Alembic's ``EnvironmentContext.configure()`` method. As is standard for all Flask extensions, Flask-Migrate can be initialized using the ``init_app`` method as well.
+The two arguments to ``Migrate`` are the application instance and the Flask-SQLAlchemy database instance. The ``Migrate`` constructor also takes additional keyword arguments, which are passed to Alembic's ``EnvironmentContext.configure()`` method. As is standard for all Flask extensions, Flask-Migrate can be initialized using the ``init_app`` method as well::
+
+    from flask_migrate import Migrate
+
+    migrate = Migrate()
```

Comment:
You'd want the `db` also initialized up here.

    db = SQLAlchemy()
    migrate = Migrate()